CorsixTH/Lua/languages/english.lua

```diff
@@ -129,6 +129,7 @@ menu_charts = {
 
 menu_debug = {
   jump_to_level               = "  JUMP TO LEVEL  ",
+  connect_debugger            = "  CONNECT LUA DEBUGGER",
```

Comment:
Sorry if I'm too detailed now, but for consistency there should be two spaces after DEBUGGER. ;-)